requirements.txt

```diff
@@ -30,7 +30,7 @@ ppgan==2.1.*
 transformers
 
 # NNCF notebook requirements
-git+https://github.com/openvinotoolkit/nncf.git#egg=nncf[tf,torch]
+git+https://github.com/eaidova/nncf.git#egg=nncf[tf,torch]
```

Comment:
Should we use this instead of the openvinotoolkit one?

Reply:
agree